docs/art/retransmit_stage.bob

```diff
@@ -0,0 +1,45 @@
+                          |  |Window Service         |                     |                             |          +---------+
+                          |  |                       |                     |          +---------------+  |          |         |
+           Packets        |  +---------+  +--------+ |                     |  Shreds  |               |  | Entries  | Replay  |
+          +----------------->+Deshred  +->+Filter  +--------------------------------->+ Blockstore    +------------>+ Stage   |
+                          |  +---------+  ++-+-----+ |                     |          |               |  |          |         |
++--------+                |  |             ^ |       |                     |          |               |  |          +---------+
+|        | Leader Schedule|  |             | |       v                     |          |               |  |
+| Bank   +---------------------------------- |     +-+---------+           |          ++--------------+  |
+|        |                |  |               |     | Repair    |           |           |                 |
++--------+    +----------------------------------->+ Service   | Incomplete|Slots      |                 |
+              |           |  |               |     |           +<----------------------+                 |
+              |           |  |               |     +-----------+           |                             |
+              |           |  |               |                             |                             |
+              |           |  +---------------------------------------------+                             |
+              |           |                  |Shreds                                                     |
+              |           |                  v                                                           |
+              v           |          +-------+---------+                                                 |
+     +--------+-----+     |          |                 |                                                 |
+     |              |     |          | Retransmitter   |                                                 |
+     | Peer         |     |          |                 |                                                 |
+     | Validators   +<---------------+                 |                                                 |
+     |              |     |          +--------+--------+                                                 |
+     |              |     |                   ^                                                          |
+     +--------------+     |                   |                                                          |
+                          +------------------------------------------------------------------------------+
+                                              |
+                                    Retransmit|
```

Comment:
Everywhere else in this diagram you label the arrows with the data type, which I think is ideal.  Here you write "Retransmit peers". Seems weird.  What are you trying to say?

Reply:
Ah. I wasn't using types intentionally, just try to label what is sent. Here, the retransmitter is getting the send information for the peers it needs to retransmit to from gossip.

Reply:
Maybe call it `PeerInfo`?  Unless the code already has a name for it?

Reply:
`ContactInfo` perhaps